src/containers/repo.jsx

```diff
@@ -500,6 +501,7 @@ class Repo extends Component {
     this.labelRGBA = this.labelRGBA.bind(this);
     this.defaultStory = this.defaultStory.bind(this);
     this.handleUpdateAllMasks = this.handleUpdateAllMasks.bind(this);
+    this.handleOpacityChange = this.handleOpacityChange.bind(this);
```

Comment:
All methods must be bound to `this` before we pass them down to child components. This is an annoying "gotcha" to allow the methods to call this container's own methods such as`this.setState`.